### PR TITLE
VAM-174 Meta boiler

### PR DESCRIPTION
**Pull request by sidjtd**
Done! (I hope)
Once its merged, simply : 

npm run make -- rpcName yourNewNameOfChoice
(ie : npm run make -- orwdal32Def orderThoseBespokenDefinitions)

2nd param optional

### CODE REVIEW

**Comment by mfuroyama** [comment]
Ooh, Mr. Pedantic here again: totally not a big deal, but let's try to name our branches with the *VAM-<<NUMBER>>-something-or-other-description* naming scheme...

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# metaBoiler
```
Comment (mfuroyama) [inline scripts/README.md]: 👍

File: scripts/metaBoiler.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+const path = require('path');
+const json = require('./rpcNameVariants.json');
+const boiler = require('./boilerTemplates');
+
+const userInput = process.argv[2] || '_';
+let myName = process.argv[3] || '';
+userInput.trim();
+
+/* ---------  Helper Functions  --------- */
+const toKebabCase = (str => str.replace(/([a-z])([A-Z])/g, '$1-$2'));
```
Comment (mfuroyama) [inline scripts/metaBoiler.js]: Consider using the lodash `kebabCase` method: https://lodash.com/docs/4.17.4#kebabCase

File: scripts/README.md (hunk 1 above)
Comment (sidjtd) [reply scripts/README.md]: Will do that from now on. I started this branch before the issue was made, should have just swapped it. Everything from now I'll make sure an issue is made for it first so we get credit

File: scripts/metaBoiler.js (hunk 2 above)
Comment (sidjtd) [reply scripts/metaBoiler.js]: it seems like lodash just has absolutely everything.... Im amazed!

File: scripts/README.md (hunk 1 above)
Comment (mfuroyama) [reply scripts/README.md]: Ahh, yeah, you're totally right! My bad on that!

**Pull request merged**